addons/extensionservice/org.openhab.extensionservice.marketplace/README.md

```diff
@@ -0,0 +1,61 @@
+# Eclipse marketplace Extension service
+
+This is an `ExtensionService`, which accesses the Eclipse IoT Marketplace and makes its content available as
+extensions.
+
+The internally used http address is
+"https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/api/p?client=org.eclipse.smarthome".
```

Comment:
Second line needed here?

Reply:
To have the link in its own row, I thought. Should I join instead?

Reply:
I would join here. The sentence is not that long, so it is quite easy to overview changes.